### DISCUSSION TITLE
[Feat]: Scroll carousel with touchpad?

### DISCUSSION THREAD

**Discussion by chertik77** [davidjerleke/embla-carousel#1022] [Help] [answered]
### Which variants of Embla Carousel are relevant to this feature request?

- [ ] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### Feature description

Placing two fingers on the laptop / desktop touchpad, moving horizontally triggers next slide or prev slide depending on the direction (left or right)

![image](https://github.com/user-attachments/assets/bdd272b8-f0cb-4af3-a221-c3d71c7bae5b)

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues
- [X] I agree to follow this project's Contributing Guidelines for feature requests

**Comment by davidjerleke** [comment, marked as answer]
@chertik77 are you looking [for this](https://www.embla-carousel.com/plugins/wheel-gestures/)?

**Reply by davidjerleke** [reply]
@chertik77 did you see my response?

**Reply by chertik77** [reply]
> @chertik77 are you looking [for this](https://www.embla-carousel.com/plugins/wheel-gestures/)?

Hey, sorry for late reply! I'll try that plugin, thank you so much 🙂
